plugins/inputs/chrony/chrony.go

```diff
@@ -38,6 +40,24 @@ func (*Chrony) SampleConfig() string {
 	return sampleConfig
 }
 
+// dialUnix opens an unixgram connection with chrony
+func dialUnix(address string) (*net.UnixConn, error) {
+	base, _ := path.Split(address)
+	local := path.Join(base, fmt.Sprintf("chronyc.%d.sock", os.Getpid()))
+	conn, err := net.DialUnix("unixgram",
+		&net.UnixAddr{Name: local, Net: "unixgram"},
+		&net.UnixAddr{Name: address, Net: "unixgram"},
+	)
+	if err != nil {
+		return nil, err
+	}
+	//socket.c:bind_unix_address
+	if err := os.Chmod(local, 0666); err != nil { // #nosec G302: needs permission to the directory too
+		return nil, err
+	}
```

Comment:
Why do "others" need to get access? Even write access?

Reply:
TBH, I have no idea why, The chrony C client does the same. If you remove the chmod or only have 664/644 it doesn't work. If you have a reason/fix, I'd be happy to hear it :-)

Reply:
Is there a way to bring this up in the chrony project? TBH: I'm a bit hesitant to move this into the code where it will make up for a reasonable source of trouble in the future and poses a security issue as we now open up the communication to chrony for everyone...

Reply:
Let me change the documentation and get back to you

Reply:
fixed; thanks!